modules/internal/ChapelRange.chpl

```diff
@@ -808,11 +859,10 @@ module ChapelRange {
   {
```

Comment:
Let's remove as much of the above comment as is no longer applicable or specific to this function.